contracts/erc20/contracts/src/LibERC20Token.sol

```diff
@@ -91,6 +92,21 @@ library LibERC20Token {
         _callWithOptionalBooleanResult(token, callData);
     }
 
+    /// @dev Retrieves the number of decimals for a token.
+    ///      Returns `18` if the call reverts.
+    /// @return The number of decimals places for the token.
+    function decimals(address token)
```

Comment:
There seems to be some confusion about whether `decimals` is part of the standard (I think it is), so I added this just to be safe.